libafl_libfuzzer/libafl_libfuzzer_runtime/src/lib.rs

```diff
@@ -215,7 +216,7 @@ macro_rules! fuzz_with {
             // Create a stacktrace observer
             let backtrace_observer = BacktraceObserver::new(
                 "BacktraceObserver",
-                unsafe { &mut BACKTRACE },
+                OwnedRefMut::owned(None),
```

Comment:
@addisoncrump can you doublecheck that this is correct? I didn't see any reason to use a static variable here(?) Nobody else seems to access it(?)
Would it be faster than having the value on the heap?

Reply:
Slow response here, sorry. IIRC that is an artifact of copy/pasting from an old example. This update seems reasonable.